.gitignore

```diff
@@ -10,6 +10,8 @@
 **/beta_*.sh
 **/.DS_Store*
 
+/privateTemplates/
```

Comment:
>privateTemplates [](start = 1, length = 16)

How about just "templates". I actually hope to see that in the future any SDK will be generated this same way

Reply:
We could add/change/rename/remove folders quite easily now.

Reply:
I'm going to keep it named after our current intent for now.